src/components/Sidebar.tsx

```diff
@@ -17,18 +17,19 @@ export const Sidebar: React.FC = () => {
   const [navItems, setNavItems] = useState<NavItemFields[]>([]);
 
   const { data: userData, isLoading: isUserLoading } = api.user.currentUser.useQuery();
-  const { pathname } = useRouter();
+  const { asPath } = useRouter();
```

Comment:
there was a bug when clicking on the user profile did not cause the green highlight on the sidebar to appear next to the Profile link. this fixed it.